### ISSUE TITLE
I'm only interested in the English version of a manga. How do I prevent the app from redownloading in other languages? (from comick)

### ISSUE THREAD

**Issue by mexteqq** [HDoujinDownloader/HDoujinDownloader#417]
How can I prevent download any other language but English from comick.io?

**Comment by HDoujinDownloader** [comment]
If you're adding URLs to the download queue manually, you can right-click the item in the download queue, select "Choose chapters", and only select the ones you want (i.e. the English chapters).

But from the way your question is worded, I assume you're using bookmarks, and it's detecting the non-English chapters as new ones.

Because this is an issue that comes up across many different websites, I'm going to add a global "preferred chapter language" setting. I'll try to get that implemented as soon as possible!

**Comment by mexteqq** [comment]
You are correct. I want to "start" using bookmarks and noticed that issue, where its downloading all languages. Or at least from comick. Having a global language preference would be great. Thank you.
